# SPDX-FileCopyrightText: Alliander N. V.
#
# SPDX-License-Identifier: Apache-2.0

fr3_arm_controller:
  ros__parameters:
    type: joint_trajectory_controller/JointTrajectoryController
    command_interfaces:
      - position
    state_interfaces:
      - position
      - velocity
    joints:
      - fr3_joint1
      - fr3_joint2
      - fr3_joint3
      - fr3_joint4
      - fr3_joint5
      - fr3_joint6
      - fr3_joint7

gripper_action_controller:
  ros__parameters:
    type: position_controllers/GripperActionController
    action_monitor_rate: 20.0
    allow_stalling: false
    goal_tolerance: 0.01
    joint: "fr3_finger_joint1"
    max_effort: 0.0
    stall_timeout: 1.0
    stall_velocity_threshold: 0.001
